dftimewolf/lib/processors/gce_forensics_vm.py

```diff
@@ -151,12 +151,15 @@ def Process(self) -> None:
         evidence_disk=None,
         platform='gcp'))
 
-    for d in [d.name for d in self.state.GetContainers(containers.GCEDisk)]:
-      self.logger.info(f'Attaching {d} to {analysis_vm_name}')
-      self.analysis_vm.AttachDisk(compute.GoogleComputeDisk(
-          self.project.project_id,
-          self.project.default_zone,
-          d))
+    disks = self.state.GetContainers(containers.GCEDiskEvidence)
+
+    for d in disks:
+      if d.project == self.project.project_id:
```

Comment:
Use a guard clause here to prevent another indentation.

```suggestion
      if d.project != self.project.project_id:
        continue
```